content/tutorials/kit-assembly.md

```diff
@@ -39,22 +46,23 @@ Make sure to read all these **before** you start assembly
 
 - Colour coding is key; try to make sure that you use RED for wires connected to a powered rail (say 12V or 5V), BLACK for wires connected to ground (0V rail) and BLUE for motors, although it is permissible to use other colours in this case if they were already soldered onto the motors provided in the kits.
 
+- DO NOT TOUCH THE MICRO SD CARD IN THE RASPBERRY PI
```

Comment:
I'm not sure this warning is necessary. If we don't bring attention to it, they _shouldnt_ try and do it for themselves